dist/tools/tunslip/tunslip6.c

```diff
@@ -147,18 +158,34 @@ stamptime(void)
 int
 is_sensible_string(const unsigned char *s, int len)
 {
-    int i;
+    if(len > 0) {
+        if (s[0] == 0x60) {
+            /* Possibly IPv6 packet with default traffic class, assume non-printable */
+            return 0;
+        }
+    }
 
-    for (i = 1; i < len; i++) {
+    int ret = 0;
+    for (int i = 1; i < len; i++) {
         if (s[i] == 0 || s[i] == '\r' || s[i] == '\n' || s[i] == '\t') {
             continue;
         }
         else if (s[i] < ' ' || '~' < s[i]) {
             return 0;
         }
+        /* only return 1 if the string contains at least one letter or number */
```

Comment:
It's not, it's requiring at least one letter or number before it can accept the given string as printable. It's there to avoid misinterpreting parts of packets as "strings" such as mistaking
```
"`'  \r \0 "^~\n"
```
for a valid string and printing it instead of leaving it in the packet buffer.
I reasoned that by requiring at least one letter or number in a string then we at least reduce the risk somewhat.